src/examples/moghadas/model.py

```diff
@@ -65,13 +65,36 @@ def get_model(self, xs: np.ndarray, _, ps: dict, cm: np.ndarray) -> np.ndarray:
 
         return self.get_array_from_dict(comp_dict=model_eq_dict)
 
-    def get_hospitalized(self, solution: np.ndarray) -> np.ndarray:
-        idx = self.c_idx["i_h"]
-        idx_2 = self.c_idx["q_h"]
-        idx_3 = self.c_idx["h"]
-        return self.aggregate_by_age(solution, idx) + \
-            self.aggregate_by_age(solution, idx_2) + self.aggregate_by_age(solution, idx_3)
+    def get_infected(self, solution: np.ndarray) -> np.ndarray:
+        """
+        All individuals not in s, r, d, i, hosp, icu are counted as infected
+        """
+        total = solution.sum(axis=1)
+        s = self.aggregate_by_age(solution, self.c_idx["s"])
+        r = self.aggregate_by_age(solution, self.c_idx["r"])
+        d = self.aggregate_by_age(solution, self.c_idx["d"])
+        i = self.aggregate_by_age(solution, self.c_idx["i"])
```

Comment:
Rename this compartment to `inf` - then it will be aligned with the other naming conventions.